tests/components/switcher_kis/test_cover.py

```diff
@@ -182,6 +188,174 @@ async def test_cover_control_fail(hass: HomeAssistant, mock_bridge, mock_api) ->
             )
 
         assert mock_api.call_count == 4
-        mock_control_device.assert_called_once_with(27)
+        mock_control_device.assert_called_once_with(27, 0)
         state = hass.states.get(ENTITY_ID)
         assert state.state == STATE_UNAVAILABLE
+
+
+@pytest.mark.parametrize("mock_bridge", [[DEVICE2]], indirect=True)
+async def test_cover2(
```

Comment:
This test is an exact copy of the `test_cover` with only change in the `entity_id`, please parameterize the `entity_id` in the existing `test_cover` test